### PR TITLE
add(epp): redis sentinel proc

### PR DESCRIPTION
**Pull request by garnier-quentin**
## Description

-

## Target serie

- [X] 20.04.x
- [X] 20.10.x
- [X] 21.04.x
- [X] 21.10.x
- [X] 22.04.x (master)

### CODE REVIEW

File: en/integrations/plugin-packs/procedures/applications-redis-sentinel.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+| cluster.sentinels.objectively_down.count                                                    | Number of objectively down sentinel instances  |      |
+| quorum status                                                                               | Status of sentinel voted quorum                |      |
+| status                                                                                      | Status of sentinel instance                    |      |
+| *cluster_name~sentinel_address:sentinel_port*#cluster.sentinel.ping_ok.latency.milliseconds | Last ok ping latency                           | ms   |
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Prerequisites
+
+To control your Redis Sentinel, the Poller should be able to perform ```redis-cli``` requests over TCP/26379 port (by default).
```
Comment (cg-tw) [inline en/integrations/plugin-packs/procedures/applications-redis-sentinel.md:57]:
```suggestion
To control your Redis Sentinel, the Poller should be able to perform **redis-cli** requests over the TCP/26379 port (by default).
```

File: en/integrations/plugin-packs/procedures/applications-redis-sentinel.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+3. On the Centreon Web interface, install the **Redis Sentinel** Centreon Pack on the **Configuration > Plugin Packs** page.
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Configuration
+
+### Host
+
+* Log into Centreon and add a new Host through **Configuration > Hosts**
+* Fill the **Name**, **Alias** & **IP Address / DNS** fields according to your *Redis Sentinel* server settings
```
Comment (cg-tw) [inline en/integrations/plugin-packs/procedures/applications-redis-sentinel.md:101]:
```suggestion
* Fill the **Name**, **Alias** & **IP Address / DNS** fields according to your **Redis Sentinel** server settings
```

File: en/integrations/plugin-packs/procedures/applications-redis-sentinel.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+* Select the *applications-redis-sentinel-custom* template to apply to the Host
+* Once the template applied, some Macros marked as 'Mandatory' hereafter have to be configured.
```
Comment (cg-tw) [inline en/integrations/plugin-packs/procedures/applications-redis-sentinel.md:103]:
```suggestion
* Once the template is applied, fill in the corresponding macros. Some macros are mandatory.
```

File: en/integrations/plugin-packs/procedures/applications-redis-sentinel.md [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+
+| Mandatory | Name             | Description                                                                  |
+|:----------|:-----------------|:-----------------------------------------------------------------------------|
+|           | SENTINELPORT     | (Default: '26379')                                                           |
+|           | SENTINELUSERNAME | Sentinel username (redis-cli >= 6.x mandatory)                               |
+|           | SENTINELPASSWORD | Sentinel password                                                            |
+|           | EXTRAOPTIONS     | Any extra option you may want to add to the command (eg. a --tls --insecure) |
+
+## How to install _redis-cli_ 6.x ?
```
Comment (cg-tw) [inline en/integrations/plugin-packs/procedures/applications-redis-sentinel.md:112]:
```suggestion
## How to install redis-cli 6.x ?
```

File: en/integrations/plugin-packs/procedures/applications-redis-sentinel.md [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+
+To use _tls_ and/or ACL users, you need _redis-cli_ >= 6.x.
```
Comment (cg-tw) [inline en/integrations/plugin-packs/procedures/applications-redis-sentinel.md:114]:
```suggestion
To use TLS and/or ACL users, you need **redis-cli** >= 6.x.
```

File: en/integrations/plugin-packs/procedures/applications-redis-sentinel.md [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+
+```bash
+yum install epel-release
+yum install http://rpms.remirepo.net/enterprise/remi-release-7.rpm 
+yum --enablerepo=remi install redis
+```
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## How to check in the CLI that the configuration is OK and what are the main options for ?
```
Comment (cg-tw) [inline en/integrations/plugin-packs/procedures/applications-redis-sentinel.md:128]:
```suggestion
## How to check in the CLI that the configuration is OK and what are the main options for? 
```

File: fr/integrations/plugin-packs/procedures/applications-redis-sentinel.md [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+| cluster.sentinels.slaves.detected.count                                                     | Number of detected sentinels                   |      |
+| cluster.sentinels.subjectively_down.count                                                   | Number of subjectively down sentinel instances |      |
+| cluster.sentinels.objectively_down.count                                                    | Number of objectively down sentinel instances  |      |
+| quorum status                                                                               | Status of sentinel voted quorum                |      |
+| status                                                                                      | Status of sentinel instance                    |      |
+| *cluster_name~sentinel_address:sentinel_port*#cluster.sentinel.ping_ok.latency.milliseconds | Last ok ping latency                           | ms   |
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Prerequisites
```
Comment (cg-tw) [inline fr/integrations/plugin-packs/procedures/applications-redis-sentinel.md:55]:
```suggestion
## Prérequis
```

File: fr/integrations/plugin-packs/procedures/applications-redis-sentinel.md [hunk 8]
```diff
@@ -0,0 +1,179 @@
...
+
+Afin de superviser Redis Sentinel, le collecteur doit pouvoir réaliser des requêtes avec l'utilitaire ```redis-cli``` sur le port TCP/26379 (par défaut).
```
Comment (cg-tw) [inline fr/integrations/plugin-packs/procedures/applications-redis-sentinel.md:57]:
```suggestion
Afin de superviser Redis Sentinel, le collecteur doit pouvoir réaliser des requêtes avec l'utilitaire **redis-cli** sur le port TCP/26379 (par défaut).
```

File: fr/integrations/plugin-packs/procedures/applications-redis-sentinel.md [hunk 9]
```diff
@@ -0,0 +1,179 @@
...
+3. Sur l'interface Web de Centreon, installer le Pack **Redis Sentinel** depuis la page **Configuration > Packs de plugins**.
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Configuration
+
+### Hôte
+
+* Ajoutez un Hôte à Centreon depuis la page **Configuration > Hôtes**
+* Complétez les champs **Nom**,**Alias** & **IP Address / DNS** correspondant à votre serveur **Redis Sentinel**.
```
Comment (cg-tw) [inline fr/integrations/plugin-packs/procedures/applications-redis-sentinel.md:101]:
```suggestion
* Complétez les champs **Nom**, **Alias** & **IP Address / DNS** correspondant à votre serveur **Redis Sentinel**.
```

File: fr/integrations/plugin-packs/procedures/applications-redis-sentinel.md [hunk 10]
```diff
@@ -0,0 +1,179 @@
...
+* Appliquez le Modèle d'Hôte **applications-redis-sentinel-custom**
+
+* Une fois le modèle appliqué, les Macros ci-dessous indiquées comme requises (*Mandatory*) doivent être renseignées
```
Comment (cg-tw) [inline fr/integrations/plugin-packs/procedures/applications-redis-sentinel.md:104]:
```suggestion
* Une fois le modèle appliqué, renseignez les macros correspondantes. Attention, certaines macros sont obligatoires ("mandatory").
```

File: fr/integrations/plugin-packs/procedures/applications-redis-sentinel.md [hunk 11]
```diff
@@ -0,0 +1,179 @@
...
+* Une fois le modèle appliqué, les Macros ci-dessous indiquées comme requises (*Mandatory*) doivent être renseignées 
+
+| Mandatory | Name             | Description                                                                  |
+|:----------|:-----------------|:-----------------------------------------------------------------------------|
+|           | SENTINELPORT     | (Default: '26379')                                                           |
+|           | SENTINELUSERNAME | Sentinel username (redis-cli >= 6.x mandatory)                               |
+|           | SENTINELPASSWORD | Sentinel password                                                            |
+|           | EXTRAOPTIONS     | Any extra option you may want to add to the command (eg. a --tls --insecure) |
+
+## Comment installer _redis-cli_ 6.x ?
```
Comment (cg-tw) [inline fr/integrations/plugin-packs/procedures/applications-redis-sentinel.md:113]:
```suggestion
## Comment installer redis-cli 6.x ?
```

File: fr/integrations/plugin-packs/procedures/applications-redis-sentinel.md [hunk 12]
```diff
@@ -0,0 +1,179 @@
...
+
+Pour le support _tls_ et des utilisateurs ACLs, une version 6.x minimum de _redis-cli_ est nécessaire.
```
Comment (cg-tw) [inline fr/integrations/plugin-packs/procedures/applications-redis-sentinel.md:115]:
```suggestion
Pour le support TLS et des utilisateurs ACLs, une version 6.x minimum de **redis-cli** est nécessaire.
```

File: fr/integrations/plugin-packs/procedures/applications-redis-sentinel.md [hunk 13]
```diff
@@ -0,0 +1,179 @@
...
+```bash
+/usr/lib/centreon/plugins/centreon_redis_sentinel.pl \
+    --plugin=apps::redis::sentinel::plugin \
+    --list-mode
+ ```
+
+### Diagnostic des erreurs communes
+
+Rendez-vous sur la [documentation dédiée](../tutorials/troubleshooting-plugins.html)
+pour le diagnostique des erreurs commununes des Plugins Centreon.
```
Comment (cg-tw) [inline fr/integrations/plugin-packs/procedures/applications-redis-sentinel.md:179]:
```suggestion
pour le diagnostic des erreurs communes des Plugins Centreon.
```

File: en/integrations/plugin-packs/procedures/applications-redis-sentinel.md (hunk 5 above)
Comment (cg-tw) [inline en/integrations/plugin-packs/procedures/applications-redis-sentinel.md:114]: Th sentence doesn't make sense and doesn't match the French version

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by cg-tw — APPROVED** [review]

**Review by Sims24 — APPROVED** [review]

**Pull request merged**